src/main/kotlin/commands/LmgtfyCommand.kt

```diff
@@ -0,0 +1,29 @@
+package commands
+
+import Command
+import arg
+import doesLater
+import greedyString
+
+object LmgtfyCommand : Command("lmgtfy") {
+    init {
+        greedyString("search term") {
+            doesLater { context ->
+                // Explicit types are necessary for type inference
+                val term: String = context arg "search term"
+                message.channel.send {
+                    embed {
+                        title = "Here you go!"
+                        description = "[Solution](https://lmgtfy.com/?q=${term.replace(" ", "+")})"
+                        color = Main.Colors.SUCCESS.color
```

Comment:
use Colors.BLUE, it can't really fail to parse a string so it doesn't make sense to use success